app/helpers/user_helper.rb

```diff
@@ -14,4 +14,23 @@ def can_edit_org?(current_user)
   def delete_user_link(user)
     govuk_button_link_to "Delete this user", delete_confirmation_user_path(user), warning: true
   end
+
+  def display_pending_email_change_banner?(user)
+    user.unconfirmed_email.present? && user.email != user.unconfirmed_email
+  end
+
+  def pending_email_change_title_text(current_user, user)
+    if current_user == user
+      "You have requested to change your email address to #{user.unconfirmed_email}."
+    else
+      "There has been a request to change this user’s email address to #{user.unconfirmed_email}."
+    end
+  end
+
+  def pending_email_change_banner_text(current_user)
+    text = "A confirmation link has been sent to the new email address. The current email will continue to work until the change is confirmed."
+    text += " Deactivating this user will cancel the email change request." if current_user.support?
```

Comment:
Can coordinators deactivate users in their org? If so, they should probably see this too.

Reply:
Yes they can, and I agree, makes sense to show it to coordinators as well!